multistore/layouts.go

```diff
@@ -1,20 +1,20 @@
-package layouts
+package multistore
 
 import (
 	"context"
 
 	"github.com/influxdata/chronograf"
 )
 
-// MultiLayoutsStore is a LayoutsStore that contains multiple LayoutsStores
+// Layouts is a LayoutsStore that contains multiple LayoutsStores
```

Comment:
should this read `Layouts is a MultiStore that contains multiple LayoutsStores`? seems tautological as-is